DB.md

```diff
@@ -0,0 +1,87 @@
+# Some notes on interacting with postgresql database
+
+## postgresql extensions
+
+```
+pip install pgxnclient
+pgxn install vector
+```
+
+## workaround for psycopg2 not finding the socket file
```

Comment:
maybe its a dumb question, but we're not using psycopg3 now ?

Reply:
done